godaddy-wordpress/woocommerce
# Add the status filter links

## Overview

This adds the different status filter links at the top of the Reviews page, with the appropriate number for each status.

## Story: [MWC-5341](https://jira.godaddy.com/browse/MWC-5341)

## QA

1. Go to Products > Reviews
    - [x] You see 5 links at the top: All, Pending, Approved, Spam, and Trash
    - [x] "All" is currently in bold.
    - [x] Each status shows the correct count for the number of reviews you have. (Note: "All" does NOT include "Spam" or "Trash"; it's basically a combination of "Pending" + "Approved".)
2. Click through each status.
    - [x] The `comment_status` query arg in the URL changes to the correct value.
    - [x] The status you clicked becomes bold.
    - [x] The results in the table update accordingly.

~**Note: It's normal that the actual results in the table do not change, as that is not part of this story.**~ **EDIT:** Results will now change.
